src/next-collection-view.js

```diff
@@ -577,6 +577,17 @@ const CollectionView = Backbone.View.extend({
     this.Dom.appendContents(this.el, els, {_$el: this.$el});
   },
 
+  swapChildViews(view1, view2) {
+    if (!this.children.hasView(view1) || !this.children.hasView(view1)) {
```

Comment:
I think it'd be nice to be able to swap out views that don't belong to the same parent. But an issue I foresee is swapping views and the collection view has a filter that runs and it creates some type of unintended side affect.